src/main/java/it/pn/frontend/e2e/pages/mittente/DisserviziAppPAPage.java

```diff
@@ -129,6 +161,23 @@ public void checkDisserviziInCorso() {
         }
     }
 
+    public void checkDisservizioRisolto() {
+        aggionamentoPagina();
+        List<WebElement> disserviziTableRows = disserviziTable.findElements(By.id("tableDowntimeLog.row"));
+        if (!disserviziTableRows.isEmpty()) {
+            WebElement primaRiga = disserviziTableRows.get(0);
+            WebElement dataFinePrimaRiga = primaRiga.findElements(By.xpath("//td[@data-testid='tableDowntimeLog.row.cell']//div//div//p[contains(text(), 'ore')]")).get(1);
+            WebElement statoPrimaRiga = primaRiga.findElement(By.xpath("//td[@data-testid='tableDowntimeLog.row.cell']//div//div//span[contains(text(), 'Risolto')]"));
+            if (dataFinePrimaRiga.isDisplayed() && statoPrimaRiga.isDisplayed()) {
+                logger.info("Disservizio risolto trovato");
+            } else {
+                logger.error("Non si visualizza un record in elenco relativo ad un disservizio risolto");
+                Assert.fail("Non si visualizza un record in elenco relativo ad un disservizio risolto");
+            }
+        }
+
+    }
+
```

Comment:
Same with what said before, check if there is one already.

Reply:
I should find that the first row of the column has resolved the dissservice

Reply:
i need check only first row
is same??